card.go

```diff
@@ -24,8 +24,8 @@ type CardFunding string
 // For more details see https://stripe.com/docs/api#create_card and https://stripe.com/docs/api#update_card.
 type CardParams struct {
 	Params
-	Token                                         string
-	Customer, Recipient                           string
+	External_Account, Token                       string
```

Comment:
How would you handle this? From that link it tells me it's either a token or a dictionary (similar to the current implementation for customer). Second, why is ```AppendDetails``` using card instead of ```source```?